.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: The clearer your bug report, the faster it will be fixed!
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Code to reproduce the error**
+The simplest code snippet that produces your bug.
+
+**Error logs (if any)**
+Provide error logs if there are any.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Packages version:**
+Run `pip freeze | grep smolagents` and paste it here.
```

Comment:
Maybe we will need to add info about other packages. We can add in future PRs, as we need them.